Jenkinsfile

```diff
@@ -1 +1,10 @@
+node('linux') {  
```

Comment:
You are using a "scripted pipeline" here which is not performing any source code checkout per default.
pre-commit is providing a hint about missing sources in its error message "An error has occurred: FatalError: git failed. Is it installed, and are you in a Git repository directory?"
The checkout can simply be performed via `checkout scm` before the `sh`step. For more details see https://jenkins.io/doc/book/pipeline/jenkinsfile/.

Reply:
Right, I focused on the "is it installed" part, thanks!